scripts/report-bundle-sizes-as-a-pr-comment.js

```diff
@@ -0,0 +1,189 @@
+const { command } = require('./lib/command')
+const { runMain, timeout } = require('./lib/execution-utils')
+const { getOrg2ApiKey, getGithubAccessToken, getOrg2AppKey } = require('./lib/secrets')
+
+const PR_COMMENT_HEADER = 'Bundles Sizes Evolution'
+const BASE_BRANCH = process.env.MAIN_BRANCH
+const LOCAL_BRANCH = process.env.CI_COMMIT_REF_NAME
+const PR_COMMENTER_AUTH_TOKEN = command`authanywhere`.run().split(' ')[2].trim()
+const GITHUB_TOKEN = getGithubAccessToken()
+const ONE_DAY_IN_SECOND = 24 * 60 * 60
+const TIMEOUT_DURATION_MS = 5000
+const FETCH_RETRIES = 4
+
+runMain(async () => {
+  const lastCommonCommit = getLastCommonCommit(BASE_BRANCH, LOCAL_BRANCH)
+  const latestLocalCommit = process.env.CI_COMMIT_SHORT_SHA
+  const prs = await getPRs(LOCAL_BRANCH)
```

Comment:
**❓ question**: If this script is running on the CI, isn't the PR available as [`process.env.CI_EXTERNAL_PULL_REQUEST_IID`](https://docs.gitlab.com/ee/ci/variables/predefined_variables.html#predefined-variables-for-external-pull-request-pipelines)? If it is, this will solve the mutiple PRs problem.

Reply:
Right, it could be easier I will try this 👍